spec/requests/catalog_request_spec.rb

```diff
@@ -140,6 +141,22 @@
     end
   end
 
+  describe "collection-level notes" do
+    let(:document_id) { "C1588" }
+    it "renders the notes on the collection show page" do
+      expect(response.body).to include("Consists primarily of three diaries that William Dundas Bathurst (1859-1940)")
+      expect(response.body).to include("No materials were removed from the collection during 2018 processing beyond")
+    end
+
+    context "when viewing the child component pages" do
+      let(:document_id) { "C1588_c15" }
+      it "renders the notes on the collection show page" do
+        expect(response.body).to include("Consists primarily of three diaries that William Dundas Bathurst (1859-1940)")
+        expect(response.body).to include("No materials were removed from the collection during 2018 processing beyond")
```

Comment:
issue: I'm seeing multiple duplicate notes in this document.
I'm guessing that isn't the behavior we are looking for. You can test for this by adding this spec here:
`expect(response.body.scan("Open for research").count).to eq 1`

thought: I wonder if we should remove dupes